src/components/views/Credentials/CredentialShare/index.tsx

```diff
@@ -22,7 +23,15 @@ export const CredentialShare: React.FC<CredentialShareProps> = (props) => {
 
   return (
     <>
-      {props.children || <Link onClick={() => setModalOpen(true)}>Share</Link>}
+      {props.children || (
+        <UnstyledButton onClick={() => setModalOpen(true)}>
+          <HoverPathFill>
+            <Box p={1}>
+              <ShareIcon />
```

Comment:
@tobek I updated this button to match design I don't think we were using a button that just said `share` anywhere but wanted to double check that I didn't override something

Reply:
Not at all, that was just a placeholder until these updates, thanks for checkin